jctools-core/src/main/java/org/jctools/queues/MpmcArrayQueue.java

```diff
@@ -215,6 +220,7 @@ public E peek() {
         return e;
     }
 
+    // $gen:ignore
```

Comment:
In theory I could work out automatically to ignore these methods based on the intersection of the MessagePassingQueue and Queue interface. i.e. all those in  Queue but not in MessagePassingQueue are ignored but there are some special cases such as the MPSC offerIfBelowThreshold and failFastOffer which are not specified on any interface. 

Let's just keep this for now